pupynere.py

```diff
@@ -526,7 +534,7 @@ def _var_metadata(self, name):
 
         # vsize
         if not var.isrec:
-            vsize = var.data.size * var.itemsize
+            vsize = var.size * var.itemsize
             vsize += -vsize % 4
```

Comment:
This is not your code, but good grief. After some thought, I think this rounds `vsize` down to the nearest multiple of 4.

Reply:
Search for "padding" in [this document](https://www.unidata.ucar.edu/software/netcdf/docs/file_format_specifications.html) and be amazed.

Reply:
:man_facepalming: